tsfile/src/main/java/org/apache/iotdb/tsfile/read/common/Path.java

```diff
@@ -18,10 +18,12 @@
  */
 package org.apache.iotdb.tsfile.read.common;
 
+import com.sun.org.apache.xerces.internal.impl.xs.util.XSInputSource;
```

Comment:
Is this a useless import? Please check, remove it if it's useless.

Reply:
Thanks for your suggestion! The useless import code is removed.